addons/mail/static/src/models/sender/sender.js

```diff
@@ -0,0 +1,141 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many, one } from '@mail/model/model_field';
+import { link, clear, insertAndReplace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'Sender',
```

Comment:
I really don't like the way this model is named.
It's a technical manager that is called like a person in the context of sending a message.
This is an arbitrary split of `Thread` for queuing message post.
Please use a prefix `Thread` to clarify that.
I strongly recommend **_not_** using "Sender" in the naming.

Name suggestion: `ThreadMessagePostQueue`.

Reply:
As you wish

Reply:
Cool, note that with this rename, now the concept "queue" is properly introduced in modelling. As a result, you could simplify method names because the record of this model is a queue by itself.